routes/api.php

```diff
@@ -30,6 +31,7 @@
     Route::resource('sender-identity', SenderIdentityController::class)->except(['create', 'edit']);
     Route::resource('campaign', CampaignController::class)->except(['create', 'edit']);
     Route::resource('roles', RoleController::class)->except(['create', 'edit']);
+    Route::resource('users', UserController::class);
```

Comment:
we have singular names in the route. Please update.

Also, the create and edit needs to be put inside `except()`